src/LiveComponent/src/Test/InteractsWithLiveComponents.php

```diff
@@ -44,4 +44,24 @@ protected function createLiveComponent(string $name, array $data = [], ?KernelBr
             self::getContainer()->get('router'),
         );
     }
+
+    protected function assertComponentEmitEvent(TestLiveComponent $testLiveComponent, string $eventName, ?array $parameters = null): void
+    {
+        $eventData = $testLiveComponent->getEmittedEvent($testLiveComponent->render(), $eventName);
```

Comment:
"event data" refers to the data passed when emitting an event, but here you are getting the event (in its arrayable form):
 
```suggestion
        $event = $testLiveComponent->getEmittedEvent($testLiveComponent->render(), $eventName);
```